### DISCUSSION TITLE
Wer braucht den yf-bootmanager?

### DISCUSSION THREAD

**Discussion by fda77** [Freetz-NG/freetz-ng#590] [Allgemein]
Würde jemand die Option von PeterPawns bootmanager im menuconfig vermissen?
Dies betrifft ausschliesslich das AVM-Webinterface. Diese ständigen Quengeleien um diese Kleinstoption nerven.
Problematisch ist dass Peter es nicht möchte wenn daran irgend etwas geändert wird. Daher ist er Bootmanager nicht 100% kompatible zu Freetz. Pullrequests werden ignoriert: https://github.com/PeterPawn/YourFritz/pull/50
Um dem ganzen ein Ende zu setzen könnte man das ganze Ding einfach rauswerfen. 
Hat jemand ausser mir ne Meinung dazu?

**Comment by Prisrak1** [comment]
frage doch, wer was überhabt benötigt? Wäre interessant es zu erfahren .. Ich für mich als Spaß nur Tor, Privoxy o. so. ..

**Reply by fda77** [reply]
Dinge die niemand benutzt werden einfach nicht aktualisiert. Es gibt genug Packages die über 10 Jahre unverändert sind oder auch gar keine Updates mehr bekommen (xmail zb, das wird aber nich genutzt)

**Comment by JosKabe** [comment]
Hallo!

Bei der 7272 ist mir aufgefallen, dass der _yf-bootmanager_ im Fritz WebGUI nicht funktioniert. Sieht dann so aus:
![yf-bootmanager_7272_fritz-webgui](https://github.com/Freetz-NG/freetz-ng/assets/145557937/e0db52c8-7980-4e84-8332-05b9b0c026a9)

Im Freetz WebGUI geht es:
![yf-bootmanager_7272_freetz-webgui](https://github.com/Freetz-NG/freetz-ng/assets/145557937/bb662836-0409-402e-974a-dbda468440fc)

Und mit den alten freetz (non ng) Images funktionierte es noch im Fritz WebGUI:
![yf-bootmanager_7272_fritz-webgui-old](https://github.com/Freetz-NG/freetz-ng/assets/145557937/dc2fe333-a28f-4e0f-b1e3-3a77f8ae0181)

Bei anderen Modellen (7430, 7490, 7560, 7580, 7590) funktioniert der yf-bootmanager dagegen im Fritz WebGUI einwandfrei.

**Reply by fda77** [reply]
Frag am besten hier nach: https://github.com/PeterPawn/YourFritz/

**Comment by freeesty** [comment]
auf meiner 7530 mit .57 verursacht der YF Bootmanager ne Reboot Schleife durch den AVM Watchguard.
Es reicht doch, wenn man im Freetz Web GUI die Partition auswählen kann, muss doch nicht noch extra im AVG Gui sein. Vor allem, wenn dadurch Probleme entstehen.

**Reply by fda77** [reply]
Falls du am BM Fehler melden möchtest mach das am besten beim Peter, ich glaube nicht dass er hier mitliest

Ich hab jetzt wirklich einen realistischen Anwendungsfall gefunden: Freetz im no-freetz Modus bauen, dh ohne alles wie Busybox oder Freetz-webif. Es werden dann nur mods aus der [fwmod_custom ](https://github.com/Freetz-NG/freetz-ng/blob/master/fwmod_custom) angewendet, 

Ich bin trotzdem noch immer dafür den BM rauszuwerfen. Damit gibt es ständig ärger und die Lizenz schränkt einen stark ein, zb dürfen Fehler nicht behoben werden obwohl es dafür schon einen Patch gibt und anderer Quatsch

**Reply by georg3003** [reply]
Eigentlich ist es eine interessante Sache, aber es geht ja per Freetz-Interface auf Port 81, also deshalb fast kann weg, nur wie kann ich von außen auf das Freetz-web-if zugreifen?

Aber gerade fällt mir da ein, daß wenn ich evtl nur eine fast unveränderte FW (mit freetz-ng aber doch ohne Erweiterungen) erstelle und installiere, ich dann doch nicht umschalten kann. deshalb besser drin lassen.

**Reply by fda77** [reply]
Die Frage wurde die letzten Tage in einer anderen Diskussion besprochen mit ein paar Möglichkeiten
Bei no-freetz gibt es kein :81
